frontend/components/wrapper/pages/CourseDetail.tsx

```diff
@@ -177,6 +179,7 @@ export function CourseDetailWrapper({
 
         setIsLiked(!isLiked)
         setGoodCount((count) => (isLiked ? count - 1 : count + 1))
+        router.refresh()
```

Comment:
```suggestion
        setIsLiked(!isLiked)
        setGoodCount(!isLiked ? goodCount + 1 : goodCount - 1)
```

状態に合わせて変えてもらいたいんですがリアルタイムでいいねを表示させなくてもいいのでお願いします。

Reply:
リアルタイムでいいねを表示させずにUIを変える形でというのはどういうことですか？

Reply:
ボタン押すたびに都度goodCountをバックエンド側に取得しにいかないってことです。